roles/junit2json/README.md

```diff
@@ -0,0 +1,308 @@
+  Merge_multiple_JUnit_XML_files_into_single_consolidated_report4-->|Task| Write_merge_resulting_file5[write merge resulting file<br>When: **junit2json merged xml stdout   length   10**]:::task
+  Write_merge_resulting_file5-->|Task| Override_the_xml_files_list_for_conversion6[override the xml files list for conversion]:::task
+  Override_the_xml_files_list_for_conversion6-->End
+```
+
+
+### Graph for expand.yml
+
+```mermaid
+flowchart TD
+Start
+classDef block stroke:#3498db,stroke-width:2px;
+classDef task stroke:#4b76bb,stroke-width:2px;
+classDef includeTasks stroke:#16a085,stroke-width:2px;
+classDef importTasks stroke:#34495e,stroke-width:2px;
+classDef includeRole stroke:#2980b9,stroke-width:2px;
+classDef importRole stroke:#699ba7,stroke-width:2px;
+classDef includeVars stroke:#8e44ad,stroke-width:2px;
+classDef rescue stroke:#665352,stroke-width:2px;
+
+  Start-->|Task| Print_path_item_value0[print path item value]:::task
+  Print_path_item_value0-->|Task| Collect_the_item_stat1[collect the item stat]:::task
+  Collect_the_item_stat1-->|Task| Update_junit2json_reports_list_with_a_JUnit_XML_report_item2[update junit2json reports list with a junit xml<br>report item<br>When: **junit2json path item stat stat exists   default<br>false**]:::task
+  Update_junit2json_reports_list_with_a_JUnit_XML_report_item2-->End
+```
+
+
+### Graph for main.yml
+
+```mermaid
+flowchart TD
+Start
+classDef block stroke:#3498db,stroke-width:2px;
+classDef task stroke:#4b76bb,stroke-width:2px;
+classDef includeTasks stroke:#16a085,stroke-width:2px;
+classDef importTasks stroke:#34495e,stroke-width:2px;
+classDef includeRole stroke:#2980b9,stroke-width:2px;
+classDef importRole stroke:#699ba7,stroke-width:2px;
+classDef includeVars stroke:#8e44ad,stroke-width:2px;
+classDef rescue stroke:#665352,stroke-width:2px;
+
+  Start-->|Task| Validate_some_variables0[validate some variables]:::task
+  Validate_some_variables0-->|Task| Initialize_input_reports_variable1[initialize input reports variable]:::task
+  Initialize_input_reports_variable1-->|Task| Initialize_reports_variable2[initialize reports variable]:::task
+  Initialize_reports_variable2-->|Include task| expand_yml3[expand the input list to list of existing files<br>include_task: expand yml]:::includeTasks
+  expand_yml3-->|Include task| merge_yml4[merge junit xml reports into single file<br>junit2json do merge true<br>When: **junit2json do merge and  junit2json found xml<br>reports matched   0**<br>include_task: merge yml]:::includeTasks
+  merge_yml4-->|Include task| convert_yml5[convert xml to json<br>When: **junit2json reports list   default       length   0**<br>include_task: convert yml]:::includeTasks
+  convert_yml5-->End
+```
+
+
+## Playbook
+
+```yml
+---
+
+- name: "Example playbook to use the role redhatci.ocp.junit2json role"
+  hosts: localhost
+  vars:
+    global_json_reports_list: []
```

Comment:
this vars line is a bad example, because the nature of passing vars to a role makes it so every task inside the role has the same var definition block, every task receives an empty list for this variable, please remove